app/src/main/java/org/sagebase/crf/CrfActivitiesFragment.java

```diff
@@ -17,90 +17,76 @@
 
 package org.sagebase.crf;
 
+import android.support.annotation.VisibleForTesting;
 import android.widget.Toast;
 
-import com.google.common.collect.Lists;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
 
-import org.researchstack.backbone.DataProvider;
 import org.researchstack.backbone.model.SchedulesAndTasksModel;
-import org.researchstack.backbone.step.InstructionStep;
-import org.researchstack.backbone.step.PermissionsStep;
-import org.researchstack.backbone.step.Step;
-import org.researchstack.backbone.task.OrderedTask;
 import org.researchstack.backbone.task.Task;
-import org.researchstack.backbone.ui.ActiveTaskActivity;
-import org.researchstack.backbone.ui.ViewTaskActivity;
 import org.researchstack.skin.ui.fragment.ActivitiesFragment;
-import org.sagebase.crf.step.CrfHeartRateCameraStep;
-import org.sagebionetworks.bridge.researchstack.CrfDataProvider;
 import org.sagebionetworks.bridge.researchstack.CrfTaskFactory;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
 /**
  * Created by TheMDP on 10/19/17
  */
-
 public class CrfActivitiesFragment extends ActivitiesFragment {
-
-    public static final String TASK_ID_HEART_RATE_MEASUREMENT = "HeartRate Measurement";
-    public static final String TASK_ID_CARDIO_12MT = "Cardio 12MT";
-    public static final String TASK_ID_STAIR_STEP = "Cardio Stair Step";
+    // Task IDs that should be hidden from the activities page. Visible to enable unit tests.
+    @VisibleForTesting
+    static final Set<String> HIDDEN_TASK_IDS = ImmutableSet.of("clinic1", "clinic2");
+
+    // Mapping from task ID to resource name. Visible to enable unit tests.
+    @VisibleForTesting
+    static final Map<String, String> TASK_ID_TO_RESOURCE_NAME =
+            ImmutableMap.<String, String>builder()
+                    .put("HeartRate Measurement", "heart_rate_measurement")
```

Comment:
I like to have one source of truth for Task IDs, can you keep the TASK_ID_HEART_RATE_MEASUREMENT around, and then reference them in the ImmutableMap?